ProofWidgets/Data/Html.lean

```diff
@@ -9,11 +9,12 @@ import Lean.PrettyPrinter.Delaborator.Basic
 import Lean.Server.Rpc.Basic
 
 import ProofWidgets.Component.Basic
+import ProofWidgets.Util
 
 /-! We define a representation of HTML trees together with a JSX-like DSL for writing them. -/
 
 namespace ProofWidgets
-open Lean Server
+open Lean Server ProofWidgets.Util
```

Comment:
```suggestion
open Lean Server Util
```

Reply:
I avoided this because I didn't want to open `Lean.Util` or `Lean.Server.Util`